src/components/backdrop/backdrop.scss

```diff
@@ -6,10 +6,11 @@ md-backdrop {
 
   position: absolute;
   top:0;
-  bottom:0;
   left: 0;
   right: 0;
 
+  height: 100%;
```

Comment:
It shouldn't affect any other components, because it's still in relative and only dynamic to the parent height.